### ISSUE TITLE
Cloaks don't work!

### ISSUE THREAD

**Issue by LeonusDH** [mchorse/emoticons#33]
Cloaks become invisible if the mod is installed.

**Comment by mchorse** [comment]
Yeah, there is nothing I can do about it. You can disable non-emote animations by going to my mod options (Ctrl + 0 in the world), Emoticons and then enable `Disable non-emote animations`. That's all I can help you with.
